### ISSUE TITLE
Make hashtags field customizable

### ISSUE THREAD

**Issue by willemarcel** [openstreetmap/iD#8037]
On Mapathons it's common to ask the mappers to add a hashtag to the changeset comment to make it possible to compute stats of a specific event, but many users edit the hashtags field instead of the comment and the tools used to query changesets don't support the hashtags field.

A suggestion is to customize the behaviour of the hashtags field, making it possible to hide it by the API. Other possibility is to hide it by default if iD was started with a value to the hashtags field.

**Comment by bhousel** [comment]
So I originally added the hashtags field in 2015 ,  this is the tracking issue: #2834.  We did this for mapathons, and made it so that the task manager could prefill the comments and hashtags independently if needed.

If in 5 years the "tools used to query changesets don't support the hashtags field" - can we just agree that it's a failed experiment and remove the field entirely?

It seems silly to me to add _more_ code to make "removal of the field customizable" if the `hashtags` tag never gained acceptance and nobody uses it.  Just remove it and simplify the code.

**Comment by willemarcel** [comment]
OSMCha supports querying by any changeset metadata field currently, but we don't have a complete OSM database.

The software used by MissingMaps/HOT to provide stats uses only the changeset comment though (https://github.com/AmericanRedCross/osm-stats/issues/48). Someone who needs a complete history stats would rely on the comment field too.

**Comment by quincylvania** [comment]
@bhousel Thanks for the background info. If the hashtags field is really causing more headache than benefit I'd be happy to remove it and streamline the save screen.

@willemarcel Are there times where people actually need this field, or do you think we can remove it?

**Comment by willemarcel** [comment]
@quincylvania I don't know any case people use it. I'll ask on the HOT Slack.

 I saw that JOSM is also adding the hashtags field, the same way as iD, but it's not so visible to users. Maybe it would be good to coordinate with JOSM developers if we remove it or make it only editable on the fields table.

**Comment by 1ec5** [comment]
In the past, I’ve occasionally found hashtags useful for spotting an organized mapping activity, such as a classroom activity or homework assignment. People seemed to be entering them in spontaneously, identifying their class, even if it wasn’t part of an instruction. I don’t think we’d get that spontaneous hashtagging if the field is off by default, but at the same time I don’t know how much of a benefit it has been in the grand scheme of things.

**Comment by tyrasd** [comment]
As far as I know, the HOT tasking manager is the main context where the hashtags are set consistently. Very occasionally I also see single mappers using it as a personal "categorization" of their own changesets, which is also a fine use case for it. And some organized mapping activities of companies do also seem to use it…

Also, in the meantime some more tooling does have support for it, e.g. https://ohsome-now.heigit.org/dashboard supports it actively. 

> add a hashtag to the changeset comment

I think the `hashtags` changeset-tag is too established by now to change it back to freetext changeset-comment hashtags.

> hide it by the API.

I don't think the upload UI is too overloaded for this to be of much of an issue. And I think the mapper should be aware if a tool they use pre-set the changeset's `hashtags` tags, and allow them to modify it to their needs if they want.

So, in summary, I think the benefits of removing the feature are not outweighing the small added complexity on the upload screen: let's keep it.
